### PR TITLE
Updated README.md file (removed embedded --help command output + fixed small spelling mistakes)

### PR DESCRIPTION
**Pull request by swissbuechi**
Follow-up for PR: https://github.com/Homebrew/homebrew-autoupdate/pull/114

* Removed embedded `--help` command output and inserted a link to the auto updated documentation: [Homebrew Documentation autoupdate subcommand](https://docs.brew.sh/Manpage#autoupdate-subcommand-interval-options)
* Added an example and recommended `start` command.
* Fixed some small spelling mistakes

### CODE REVIEW

**Comment by swissbuechi** [comment]
not stale

File: README.md [hunk 1]
```diff
@@ -3,14 +3,14 @@
 An easy, convenient way to automatically update Homebrew.
 
 This script will run `brew update` in the background once every 24 hours (by
-default) until explicitly told to stop, utilising `launchd`.
+default) until explicitly told to stop, utilizing `launchd`.
```
Comment (DomT4) [inline README.md:6]: Tap uses British English rather than American, so technically not a spelling issue here 😇

**Review by DomT4 — COMMENTED** [review]
Minor tweak on the intentional spelling but otherwise all good with me.

**Comment by DomT4** [comment]
Thanks again! 🎉

**Pull request merged**